### PR TITLE
MacTex 2020.0407 - Making it clear you need to restart your terminal to use

### PR DESCRIPTION
**Pull request by cpaika**
MacTex registers itself in the PATH via writing to  /etc/paths.d .  This file is only read on terminal startup, so once MaxTex is installed, I'm adding a message telling users to restart their terminal so they can use the CLI tools.

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

**Comment by cpaika** [comment]
Pushed a fix for the style problem

**Comment by vitorgalvao** [comment]
You don’t need to restart your terminal. For bash and zsh (and perhaps others) `eval "$(/usr/libexec/path_helper)"` suffices.

**Comment by cpaika** [comment]
@vitorgalvao Nice Is there a pattern for adding that to the cask install automatically, or would you rather I modify the instructions to include that instead of a restart?

**Comment by vitorgalvao** [comment]
Include both, and specify the command is for bash/zsh users.

**Comment by cpaika** [comment]
@vitorgalvao Thanks, fixed let me know what you think

File: Casks/mactex.rb [hunk 1]
```diff
@@ -75,4 +75,11 @@
                '/usr/local/texlive',
                '~/Library/texlive',
              ]
+
+  caveats <<~EOS
+    You must restart your terminal window for the installation of MacTex CLI tools to take effect.
```
Comment (vitorgalvao) [inline Casks/mactex.rb:80]: Separated into two paragraphs anyway, but we don’t double-space.

**Pull request merged**